### ISSUE TITLE
Can't rename property

### ISSUE THREAD

**Issue by borov333** [orientechnologies/orientdb#6961]
## OrientDB Version, operating system, or hardware.
- v2.2.13
## Operating System
- [x] Linux
- [x] MacOSX

## Expected behavior and actual behavior
I'm trying to rename sellCampaign property to leadCampaign. In the console the operation confirms that the property has been renamed, but in reality it didn't. See sample code below:

## Steps to reproduce the problem
```
orientdb {db=Customers}> info class LeadMetaInternal

CLASS 'LeadMetaInternal'

Records..............: 0
Default cluster......: leadmetainternal (id=145)
Supported clusters...: leadmetainternal(145), leadmetainternal_1(146), leadmetainternal_2(147), leadmetainternal_3(148)
Cluster selection....: round-robin
Oversize.............: 0.0

PROPERTIES
+----+------------+------+--------------------+---------+--------+--------+----+----+-------+----------+
|#   |NAME        |TYPE  |LINKED-TYPE/CLASS   |MANDATORY|READONLY|NOT-NULL|MIN |MAX |COLLATE|DEFAULT   |
+----+------------+------+--------------------+---------+--------+--------+----+----+-------+----------+
|0   |owner       |LINK  |Account             |true     |false   |true    |    |    |default|          |
|1   |dataState   |STRING|                    |false    |false   |false   |    |    |default|CLASSIFIED|
|2   |sellCampaign|LINK  |                    |true     |false   |true    |    |    |default|          |
|3   |category    |LINK  |SettingsLeadCategory|true     |false   |true    |    |    |default|          |
+----+------------+------+--------------------+---------+--------+--------+----+----+-------+----------+
orientdb {db=Customers}> alter property LeadMetaInternal.sellCampaign name leadCampaign                                                                                                           
Property updated successfully.

orientdb {db=Customers}> info class LeadMetaInternal                                   

CLASS 'LeadMetaInternal'

Records..............: 0
Default cluster......: leadmetainternal (id=145)
Supported clusters...: leadmetainternal(145), leadmetainternal_1(146), leadmetainternal_2(147), leadmetainternal_3(148)
Cluster selection....: round-robin
Oversize.............: 0.0

PROPERTIES
+----+------------+------+--------------------+---------+--------+--------+----+----+-------+----------+
|#   |NAME        |TYPE  |LINKED-TYPE/CLASS   |MANDATORY|READONLY|NOT-NULL|MIN |MAX |COLLATE|DEFAULT   |
+----+------------+------+--------------------+---------+--------+--------+----+----+-------+----------+
|0   |owner       |LINK  |Account             |true     |false   |true    |    |    |default|          |
|1   |dataState   |STRING|                    |false    |false   |false   |    |    |default|CLASSIFIED|
|2   |sellCampaign|LINK  |                    |true     |false   |true    |    |    |default|          |
|3   |category    |LINK  |SettingsLeadCategory|true     |false   |true    |    |    |default|          |
+----+------------+------+--------------------+---------+--------+--------+----+----+-------+----------+
orientdb {db=Customers}> 
```
Also, I noticed that if I try to update the linked class, which was mysteriously lost during the renaming of that class earlier, the CLI also confirms that the operation is successful but in the actuality the class is not linked. Check this snippet:

```
orientdb {db=Customers}> info class LeadMetaInternal

CLASS 'LeadMetaInternal'
Records..............: 0
Default cluster......: leadmetainternal (id=145)
Supported clusters...: leadmetainternal(145), leadmetainternal_1(146), leadmetainternal_2(147), leadmetainternal_3(148)
Cluster selection....: round-robin
Oversize.............: 0.0

PROPERTIES
+----+------------+------+--------------------+---------+--------+--------+----+----+-------+----------+
|#   |NAME        |TYPE  |LINKED-TYPE/CLASS   |MANDATORY|READONLY|NOT-NULL|MIN |MAX |COLLATE|DEFAULT   |
+----+------------+------+--------------------+---------+--------+--------+----+----+-------+----------+
|0   |owner       |LINK  |Account             |true     |false   |true    |    |    |default|          |
|1   |dataState   |STRING|                    |false    |false   |false   |    |    |default|CLASSIFIED|
|2   |sellCampaign|LINK  |                    |true     |false   |true    |    |    |default|          |
|3   |category    |LINK  |SettingsLeadCategory|true     |false   |true    |    |    |default|          |
+----+------------+------+--------------------+---------+--------+--------+----+----+-------+----------+
orientdb {db=Customers}> alter property LeadMetaInternal.sellCampaign LinkedClass LeadCampaign

Property updated successfully.

orientdb {db=Customers}> info class LeadMetaInternal

CLASS 'LeadMetaInternal'
Records..............: 0
Default cluster......: leadmetainternal (id=145)
Supported clusters...: leadmetainternal(145), leadmetainternal_1(146), leadmetainternal_2(147), leadmetainternal_3(148)
Cluster selection....: round-robin
Oversize.............: 0.0

PROPERTIES
+----+------------+------+--------------------+---------+--------+--------+----+----+-------+----------+
|#   |NAME        |TYPE  |LINKED-TYPE/CLASS   |MANDATORY|READONLY|NOT-NULL|MIN |MAX |COLLATE|DEFAULT   |
+----+------------+------+--------------------+---------+--------+--------+----+----+-------+----------+
|0   |owner       |LINK  |Account             |true     |false   |true    |    |    |default|          |
|1   |dataState   |STRING|                    |false    |false   |false   |    |    |default|CLASSIFIED|
|2   |sellCampaign|LINK  |                    |true     |false   |true    |    |    |default|          |
|3   |category    |LINK  |SettingsLeadCategory|true     |false   |true    |    |    |default|          |
+----+------------+------+--------------------+---------+--------+--------+----+----+-------+----------+
orientdb {db=Customers}> 
```

I tried to reload schema, restart cli, restart db, but nothing helps.

**Comment by tglman** [comment]
hi @borov333,

This is due to some refresh issue in the console, you can do a `database reload` to workaround this problem, this will be fixed in 3.0

Regards

**Comment by borov333** [comment]
Ok. I'll check it again in 3.0

**Comment by borov333** [comment]
Actually, I just renamed a property and completely restarted the database. The property still remains with the old name. How do I do the "database reload" properly as a work around?

**Comment by tglman** [comment]
hi @borov333,

That will be fixed in 3.0, does not mean is already fixed, we have the design that should resolve this problem, but the implementation is not yet done.

Regards
